packages/test-utils/src/api-client.ts

```diff
@@ -453,7 +453,7 @@ export class ApiClient {
     };
   }): Promise<AdminAgentResponse | ErrorResponse> {
     try {
-      const response = await this.axiosInstance.post("/api/admin/agents", {
```

Comment:
Does the dropping of the `api` prefix here mean that the urls are different for the old API endpoints now?  Is that going to break Kristal's admin scripts if all the URLs change suddenly?